portfolio/src/main/webapp/index.html

```diff
@@ -174,15 +174,31 @@ <h2>Photography</h2>
                 <p>You can check out more of my photography on my <a href="https://www.instagram.com/emmieohn/">Instagram</a> or <a href="https://unsplash.com/@emaohn">Unsplash</a>.</p>
             </div>
         </div>
-        <div id="comments-container"></div>
-        <div id="comment-form-container">
-            <form action="/data" method="POST">
-                <p>Name</p>
-                <input type="text" name="name" placeholder="Johnny Appleseed" />
-                <p>Comment:</p>
-                <input type="text" name="comment" />
-                <input type="submit" />
-            </form>
+        <div id="comment-section">
+            <h2>Comments</h2>
+            <div id="comments-container"></div>
+            <div id="comment-form-container">
+                <label for="request">Choose how many comments you'd like to display:</label>
+                <select name="request" id="request-selector" onchange="loadData()">
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+                </select>
```

Comment:
Optional: It seems a little odd to me to iterate possibilities like this. It might be better to not restrict the options, and instead do a free text field. You have the error handling on the conversion so your program wouldn't crash if someone types something they shouldn't. However, this might just be an html thing I'm not used to.

Reply:
You're right, an input field would be better--will change it